doc/man3/flux_msg_handler_create.adoc

```diff
@@ -59,6 +67,34 @@ are automatically sent an ENOSYS response by the dispatcher.
 stopping it.
 
 
+SECURITY
+--------
+
+As described in RFC 12, Flux connectors authenticate each broker
+connection, mapping it to a Flux userid and set of "roles" allocated
+to that user.  This userid and rolemask is stamped on all messages
+originating from the connection.
+
+A message handler only receives a message if a bit-wise and of the message
+rolemask and the message handler rolemask evaluates to a non-zero value,
+or the message rolemask contains the FLUX_ROLE_OWNER (instance owner) role.
+
+By default, the message handler rolemask is 0, but may be changed with
+`flux_msg_handler_allow_rolemask()` and `flux_msg_handler_deny_rolemask()`,
+which add or remove roles from the message handler rolemask.  The
+FLUX_ROLE_OWNER bit position is ignored in the message handler rolemask.
+
+The following roles are defined in RFC 12:
```

Comment:
The public interface in the code also defines FLUX_ROLE_NONE and FLUX_ROLE_ALL.  I think those need to be documented as well.